packages/@uppy/webcam/src/index.js

```diff
@@ -476,7 +476,11 @@ module.exports = class Webcam extends Plugin {
   }
 
   getVideo () {
-    const mimeType = this.recordingChunks[0].type
+    // Sometimes in iOS Safari, Blobs (especially the first Blob in the recordingChunks Array)
+    // have empty 'type' attributes (e.g. '') so we need to find a Blob that has a defined 'type'
+    // attribute in order to determine the correct MIME type.
+    const mimeType = this.recordingChunks.find(blob => blob.type?.length > 0).type
```

Comment:
is it safe for us to use the `blob.type?.length` optional chaining with IE11 support? @goto-bus-stop

Reply:
I have seen it elsewhere in the code (e.g. line 166 of the `packages/@uppy/webcam/src/index.js` file) so I assumed it was safe due to Uppy using transpilation during the build.